staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1177,7 +1195,8 @@ type cacheWatcher struct {
 	deadline            time.Time
 	allowWatchBookmarks bool
 	// Object type of the cache watcher interests
-	objectType reflect.Type
+	objectType    reflect.Type
```

Comment:
This field is no longer used for anything and can be removed.

Reply:
Updated, PTAL